### PR TITLE
Build VPN platform pages 

### PR DESCRIPTION
**Pull request by maxxcrawford**
## Description

Create the following 6 pages: 
- [x] Windows 10
- [x] Mac
- [x] Linux
- [x] Mobile
- [x] iOS
- [ ] ~Android~

Other tasks: 

- [x] Decouple shared button from landing page

Notes: 
- The Android VPN platform page will be added in a later PR. 

## Issue / Bugzilla link

- #10011
- #10010
- #10009
- ~#10008~ Blocked on copy right now 
- #10007 
- #10006

## Testing

Test the following URLs (including subnav, pricing links) 
- https://www-demo-pr-10013.herokuapp.com/en-US/products/vpn/desktop/linux/
- https://www-demo-pr-10013.herokuapp.com/en-US/products/vpn/desktop/mac/
- https://www-demo-pr-10013.herokuapp.com/en-US/products/vpn/desktop/windows/
- https://www-demo-pr-10013.herokuapp.com/en-US/products/vpn/mobile/
- ~https://www-demo-pr-10013.herokuapp.com/en-US/products/vpn/mobile/android/~
- https://www-demo-pr-10013.herokuapp.com/en-US/products/vpn/mobile/ios/

### CODE REVIEW

File: bedrock/products/templates/products/vpn/platforms/linux.html
Comment (maxxcrawford) [inline bedrock/products/templates/products/vpn/platforms/linux.html:11]: The section of copy that uses this variable is:

> Take an extra measure of security when you’re online — both at home and out and about — with Mozilla VPN. For a small monthly fee, it uses advanced WireGuard® protocol to encrypt your internet connection on your Linux-based computer and **four other devices**, letting you stream shows, play games, shop, and go about your daily life online knowing that you’re secure.

I have a comment with the copywriting team about making this the maximum number. It also occurs in the iOS page, and needs fixing.

File: bedrock/products/urls.py
Comment (maxxcrawford) [inline bedrock/products/urls.py:26]: Per @alexgibson, this can be removed as these are new pages.

File: l10n/en/brands.ftl
Comment (maxxcrawford) [inline l10n/en/brands.ftl:140]: This technically was only used in the Linux.ftl file, but this seems the correct place to place variables like this.

File: l10n/en/products/vpn/platforms/ios.ftl
Comment (maxxcrawford) [inline l10n/en/products/vpn/platforms/ios.ftl:17]: Revise "three more devices" to be the maximum number of devices or some other fix.

**Review by maxxcrawford — COMMENTED** [review]
Self-review. Note that the Android page is currently blocked on copy, so it has been intentionally omitted for now.

File: l10n/en/products/vpn/platforms/mac.ftl
Comment (peiying2) [inline l10n/en/products/vpn/platforms/mac.ftl:31]: should we add a `+` in in `{ $countries }+ countries` to stay consistent with Linux platform?

File: l10n/en/products/vpn/platforms/mac.ftl
Comment (peiying2) [inline l10n/en/products/vpn/platforms/mac.ftl:36]: `table` -> `tablet`?

File: l10n/en/products/vpn/platforms/mac.ftl
Comment (maxxcrawford) [reply l10n/en/products/vpn/platforms/mac.ftl:31]: I think so. There are a few other instances, but that was as approved in the copy doc.

File: bedrock/products/templates/products/vpn/platforms/ios.html
Comment (alexgibson) [inline bedrock/products/templates/products/vpn/platforms/ios.html:21]: Should utm campaign be something specific for these pages?

File: bedrock/products/urls.py
Comment (alexgibson) [inline bedrock/products/urls.py:23]: We should probably reorder these files as the first one is used for activation (same applies below)
```suggestion
             ftl_files=['products/vpn/platforms/linux', 'products/vpn/platforms/shared']),
```

**Review by alexgibson — CHANGES_REQUESTED** [review]
A couple of nits but otherwise these pages look good to me r+wc

I'll defer to Peiying on reviewing the Fluent strings.

File: bedrock/products/templates/products/vpn/platforms/ios.html
Comment (maxxcrawford) [reply bedrock/products/templates/products/vpn/platforms/ios.html:21]: The `_utm_source` above is.

Comment (maxxcrawford) [reply bedrock/products/templates/products/vpn/platforms/ios.html:21]: @rraue Do you think I should change all of these to `vpn-platform-page`?

File: bedrock/products/urls.py
Comment (maxxcrawford) [reply bedrock/products/urls.py:23]: Done!

**Review by alexgibson — APPROVED** [review]

**Pull request merged**
